RecoJets/JetProducers/python/puJetIDAlgo_cff.py

```diff
@@ -31,11 +32,49 @@
  JetIdParams = full_53x_wp,
  label = cms.string("full53x")
  )
-
+####################################################################################################################
+full_74x_chs = cms.PSet(
```

Comment:
this looks like a full copy of what's already in PileupJetIDCutParams_cfi
use 
``` from RecoJets.JetProducers.PileupJetIDCutParams_cfi import full_74x_chs``` instead